doc/org.md

```diff
@@ -127,7 +127,9 @@ the citation identifier from the database, and may optionally
 have a prefix, a locator, and a suffix. The citation key must
 begin with a letter, digit, or `_`, and may contain
 alphanumerics, `_`, and internal punctuation characters
-(`:.#$%&-+?<>~/`). Here are some examples:
+(`:.#$%&-+?<>~/`). Alternatively arbitrary citation keys can be
+given inbetween `{` and `}`. Inside them `}` and `\` can be
```

Comment:
inbetween should be two words "in between" -- or better, just "between"

Reply:
This doesn't really bring out the message that the braces are the thing to use if you have special characters not otherwise allowed.  I'd suggest something like:

```
If you want to use a citation key that do not conform to these constraints,
enclose the key in curly braces (`{` and `}`).  Inside the curly braces,
the characters `}` and `\\` must be backslash-escaped.
```

Should `{` also be escapable in braces?

Reply:
> Should { also be escapable in braces?

Everything is escapable. It's just that you only have to do it for `\` and `{`.